CONTRIBUTION.md

```diff
@@ -1,22 +1,22 @@
 # Contribution
 
 
-## Pull Requests
+## Pull Requests (PR)
 
 Your pull requests are welcome; however, they may not be accepted for various reasons.
 
 
 ### Guidelines for pull requests
 
-0. KISS - Keep It Simple, Stupid.
+0. *KISS* - Keep It Simple, Stupid.
 1. Respect Notepad++ User Manual style.
 2. Create a new branch for each PR.
 
 ## Style Guide
 
 1. First and foremost, when editing, keep it similar to what's already in that document.
 
-2. When referencing a menu item, like **File > Open**, make it bold, and use > as the separator-character between different levels.  <!-- This follows Microsoft documentation style for the separator character, so will make it more familiar to Windows users. -->
+2. When referencing a menu item, like **File > Open**, make it bold, and use *>* as the separator-character between different levels.  <!-- This follows Microsoft documentation style for the separator character, so will make it more familiar to Windows users. -->

```

Comment:
I would prefer `>` (``` `>` ```) over *>* (```*>*```), as it's trying to convey a literal character, which is what backtick syntax does; whereas italics don't carry that same meaning (and it's really hard to see an italicized angle-bracket anyway.